packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -53,14 +63,20 @@ const validator = async function (value: string) {
 
 async function canAuthenticate(accessToken: string): Promise<boolean> {
   try {
+    logger.debug("Determining if the user can be authenticated with provided access token:", accessToken);
+
+    logger.trace("Making a call to the url...");
     const response = await axios.get(`${TENDERLY_API_BASE_URL}/api/v1/user`, {
       headers: { "x-access-key": accessToken },
     });
     if (response.data.user !== undefined) {
+      logger.debug("The user can be authenticated.");
```

Comment:
```suggestion
      logger.debug("The user is not authenticated.");
```

Reply:
`if (response.data.user !== undefined)` means that user is authenticated, I changed the log to:
`User has a valid access token.`